bigtable/memorystore/src/test/java/MemcachedTest.java

```diff
@@ -130,6 +131,16 @@ public void testMemcached() throws InterruptedException {
 
     String output = bout.toString();
     assertThat(output, CoreMatchers.containsString("Value fetched from Bigtable: PQ2A.190405.003"));
-    assertThat(output, CoreMatchers.containsString("Value fetched from cache: PQ2A.190405.003"));
+
+    // retry if we didn't yet get the result in the cache
```

Comment:
Perhaps include a note about the flakiness as reason for retries

Reply:
done!